requirements/dev.txt

```diff
@@ -21,7 +21,7 @@ async-timeout==4.0.2
     # via
     #   -r requirements/validation.txt
     #   redis
-backports-zoneinfo[tzdata]==0.2.1
+backports-zoneinfo[tzdata]==0.2.1;python_version<"3.9"
```

Comment:
Why do we need this change?

Reply:
removed, this was needed on my local to run make validate